extensions/open-finder-selection-with/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "open-finder-selection-with",
+  "title": "Open Finder Selection With",
```

I'm not sure either but probably this is better for extension name. What do you think?
```suggestion
  "title": "Open With App",
```